apps/microtvm/gemmini/template_project/src/Makefile

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
It is highly recommended to use CMakeFile instead of Makefile to make it cross-platform compatible.